### PR TITLE
Correctly set call peer

### PR DESCRIPTION
**Pull request by lpradovera**
Addresses #29

### CODE REVIEW

File: spec/signalwire/relay/calling/call_spec.rb [hunk 1]
```diff
@@ -16,6 +16,23 @@
...
   end
 
+  describe 'connect_state_change' do
+    let(:peered_call){ described_class.new(client, mock_call_hash('created', 'some-call-id').dig(:params, :params, :params)) }
+
+    before do
+      client.calling.calls << peered_call
+    end
+
+    fit 'sets the peer and fires the event' do
```
Comment (bryanrite) [inline spec/signalwire/relay/calling/call_spec.rb]:
```suggestion
    it 'sets the peer and fires the event' do
```

**Review by bryanrite — CHANGES_REQUESTED** [review]
Small typo!

**Review by bryanrite — DISMISSED** [review]

**Review by bryanrite — CHANGES_REQUESTED** [review]
Add a Fix changelog entry please.

**Comment by lpradovera** [comment]
@bryanrite entry added.

**Review by bryanrite — DISMISSED** [review]

**Review by bryanrite — APPROVED** [review]

**Pull request merged**